docs/content.zh/docs/connectors/table/hive/hive_read_write.md

```diff
@@ -409,6 +409,10 @@ This configuration is set in the `TableConfig` and will affect all sinks of the
   </tbody>
 </table>
 
+### Sink Parallelism
+
+The parallelism of writing data into Hive can be configured by the corresponding table option.
+Please see the [sink parallelism]({{< ref "docs/connectors/table/filesystem" >}}#sink-parallelism) for detail.
 
```

Comment:
@lirui-apache  Would u mind helping me checking the markdown gramma of the link?

Reply:
I think we can entirely remove the sink paralleslim section. We already have a link to streaming file sink. There are many other configurations such as rolling policy, file compaction, partition commit. If we add sink parallelism section in Hive page, should we add others?

Reply:
@wuchong Good point. Actually I think `Sink Parallelism` should be a sibling, rather than a child, of `Streaming Sink`, because it applies to both batch and streaming mode, right?